src/FixedDataTable.js

```diff
@@ -410,21 +406,25 @@ var FixedDataTable = createReactClass({
      * half of the number of visible rows.
      */
     bufferRowCount: PropTypes.number,
-  },
-
-  getDefaultProps() /*object*/ {
-    return {
-      footerHeight: 0,
-      groupHeaderHeight: 0,
-      headerHeight: 0,
-      showScrollbarX: true,
-      showScrollbarY: true,
-      touchScrollEnabled: false,
-      keyboardScrollEnabled: false,
-      keyboardPageEnabled: false,
-      stopScrollPropagation: false
-    };
-  },
+  };
+
+  static defaultProps = {
+    footerHeight: 0,
+    groupHeaderHeight: 0,
+    headerHeight: 0,
+    showScrollbarX: true,
+    showScrollbarY: true,
+    touchScrollEnabled: false,
+    keyboardScrollEnabled: false,
+    keyboardPageEnabled: false,
+    stopScrollPropagation: false
+  };
+
+  constructor(props) {
+    super(props);
+    this._areColumnSettingsIdentical = this._areColumnSettingsIdentical.bind(this);
```

Comment:
You use this syntax and the blah = (args) => { body } arrow syntax.  Seems like we should choose and stick to one.

Reply:
the problem is that react-docgen can't handle static type annotated arrow functions. That throws an parser error. From my point of view using arrow functions is more elegant as it does not add the binding overhead. But as this seems not to work with static type annotations maybe we should use this syntax in general.

Reply:
Ah okay.  Could you add a "// NOTE" comment with that explanation.  Thanks for the explanation.